0007/README.md

```diff
@@ -0,0 +1,221 @@
+Since the committee's original formation, the OpenSSF has developed more
+[structure][OSSF structure] and [processes][OSSF processes] which could
+support the SLSA project.
+
+## Prior art
+
+Many governance models exist for steering or umbrella projects, i.e.; the
+current [SLSA Steering Committee], [Kubernetes Steering Council],
+numerous [other models] within the CNCF.
+
+## Goals
+
+[goals]: #goals
+
+*   High-level vision/strategy for the SLSA project should be shaped through
+    diverse inputs, including from voices outside of the SLSA project’s active
+    contributors.
+*   Transparency at all levels of SLSA governance, with clearly defined roles
+    and responsibilities.
+*   Greater recognition of existing, and future, active contributors.
+*   Strong sense of ownership for active contributors.
+*   Diverse affiliation of community leadership, such that a single entity
+    cannot unduly influence the project.
+*   Lower cost of operations for SLSA governance. The SLSA organization is
+    small, so a heavyweight process is not worth the effort. In addition, SLSA
+    can lean on the [established organizational support][OSSF structure] in the
+    OpenSSF if needed.
+
+
+## Proposed solution
+
+Below is a summary of the proposed changes, highlighting the important
+differences in **bold**. The purpose of this proposal is to get general
+agreement on the idea; exact wording will be worked out in the implementation
+PR. See [current governance doc] for reference, including the list of example
+duties for each role.
+
+### Replace “terms” / “seats” with a lightweight re-application process
+
+*   SC **no longer has 7 fixed seats**, but instead has a minimum of 3 and
+    ideally about 5.
+    *   Same guidelines for Maintainers, without the hard minimum.
+*   Individuals **self-nominate** to become a Maintainer or SC Member by
+    submitting a brief (1-2 paragraph) application consisting of:
+    *   **Evidence** of performing some of the role's duties over the recent
+        past.
+        *   For SC Member applications, this should include evidence of being
+            well-versed in SLSA.
+    *   **Statement of intent** to continue performing duties over the next
+        year.
+    *   **Affiliation** / employer.
+*   **SC Members decide** if the individual meets the desirable properties
+    (described in the next section) and respond with an official determination
+    and rationale. Decisions are effective immediately.
+    *   SC Members should attempt to reach consensus. At a minimum:
+        *   SC Member applications require explicit approval from at least
+            50% of the existing SC Members, excluding the candidate.
+        *   Maintainer applications require explicit approval from at least 2
+            existing SC members (possibly including the candidate).
+*   Existing SC Members and Maintainers **annually re-apply** for the position,
```

Comment:
When is the actual timeline?  every year on October... as an example.

Reply:
My feeling was rolling, as @abacchi said. We could have some sort of script that sends a notification when a SC member or maintainer's approval is out of date.